src/qibo/optimizers.py

```diff
@@ -56,8 +56,16 @@ def newtonian(loss, initial_parameters, method='Powell', options=None):
         options (dict): Dictionary with options accepted by
             ``scipy.optimize.minimize``.
     """
-    from scipy.optimize import minimize
-    m = minimize(loss, initial_parameters, method=method, options=options)
+    if method == 'parallel_lbfgsb':
```

Comment:
That's okay, but the fact that it is lower case may create some confusion with the equivalent pure scipy method which is `"L-BFGS-B"` with capitals and dashes. Perhaps something like `"parallel_LBFGSB"` or `"parallel_L-BFGS-B"` would be better?